src/algebra/module/pi.lean

```diff
@@ -88,7 +88,7 @@ instance distrib_mul_action' {g : I → Type*} {m : Π i, monoid (f i)} {n : Π
 { smul_add := by { intros, ext x, apply smul_add },
   smul_zero := by { intros, ext x, apply smul_zero } }
 
-lemma single_smul {α} {m : monoid α} {n : Π i, add_monoid $ f i}
+lemma single_smul {α} [monoid α] [Π i, add_monoid $ f i]
```

Comment:
Does the lemma below need to change too?

Reply:
Fixed.